packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -199,18 +199,16 @@ abstract class TextSelectionControls {
   ///
   /// This is called by subclasses when their cut affordance is activated by
   /// the user.
-  void handleCut(TextSelectionDelegate delegate, ClipboardStatusNotifier? clipboardStatus) {
+  void handleCut(TextSelectionDelegate delegate) {
     delegate.cutSelection(SelectionChangedCause.toolbar);
-    clipboardStatus?.update();
```

Comment:
It should be the delegate's job to call update on clipboardStatus since it is the owner. There may also be use case that cutSelection does not write thing into clipboard due to other short-circuited condition